### ISSUE TITLE
Add new getrpcwhitelist call

### ISSUE THREAD

**Issue by MarcoFalke** [bitcoin/bitcoin#18822]
For an RPC user it would be nice to be able to discover its whitelist (#12248). A new RPC `getrpcwhitelist` should be added for that.

#### Useful skills:

Basic understanding of the Bitcoin Core RPC interface, RPC whitelists and our functional test framework

#### Want to work on this issue?

The purpose of the `good first issue` label is to highlight which issues are suitable for a new contributor without a deep understanding of the codebase.

You do not need to request permission to start working on this. You are encouraged to comment on the issue if you are planning to work on it. This will help other contributors monitor which issues are actively being addressed and is also an effective way to request assistance if and when you need it.

For guidance on contributing, please read [CONTRIBUTING.md](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md) before opening your pull request.

**Comment by brakmic** [comment]
Interesting. Will look into it.

**Comment by promag** [comment]
Couldn't we use `help` fot this?

**Comment by MarcoFalke** [comment]
I think `help` should only return static data that can be hosted on a website or pdf

**Comment by laanwj** [comment]
No, this does not belong in `help`. It's not even related to documentation.

**Comment by promag** [comment]
Not sure if my suggestion was clear, I mean that `help` would return just the commands the user has access. But agree with Marco.

**Comment by shahramkha** [comment]
5+&&7+&

**Comment by shahramkha** [comment]
Yi cghk

**Comment by shahramkha** [comment]
- [ 44]

**Comment by david-bakin** [comment]
Is this still an issue, much less a "good first issue"?  Consider that it was addressed twice with PRs (linked) soon after it was raised, both PRs were closed by their author, one after a discussion that there should be a "generic feature discovery" framework, whatever that would be, even though _both_ had gotten some approval comments from some different people.  Status of this particular "good first issue" is very unclear to me: Should anyone be encouraged to work on it?
